pipeline/software/maya/pipe/rig_publish.py

```diff
@@ -0,0 +1,85 @@
+import maya.cmds as mc
+import os
+
+
+def publish(file_name):
+    dir_path = f"/groups/dungeons/character/Rigging/{file_name}/RigVersions"
+    link_dir_path = "/groups/dungeons/anim/rigs"
+
+    # search directory for all versions and determine new version number
+    ls_dir = os.listdir(dir_path)
```

Comment:
Rest of the project is currently using `pathlib` and `Path` objects instead of using `os.*` functions. Pathlib equivalent would be `ls_dir = dir_path.iterdir()`